app/models/miq_enterprise.rb

```diff
@@ -17,8 +17,6 @@ class MiqEnterprise < ApplicationRecord
   acts_as_miq_taggable
 
   include SupportsFeatureMixin
-  include AggregationMixin
```

Comment:
since someone's bound to ask, this relies on the has_many on hosts and vm_or_template ... which is not present and has not been for years. i can't imagine this worked.

Reply:
would be nice to keep storages in here.
just sayin'